frontend/src/main/web/lib/components/glossary/__tests__/ActionCellTest.js

```diff
@@ -0,0 +1,24 @@
+//Test disabled due to webpack and jest issue when importing component from zanata-ui
```

Comment:
Does jest have some kind of "skip" option so that it is clear in the test results that something was skipped?